docs/src/main/sphinx/release/release-352.md

```diff
@@ -0,0 +1,88 @@
+# Release 352 (XX YYY 2021)
+
+## General
+
+* Add support for [`WINDOW` clause](window_clause). ({issue}`651`)
+* Add {func}`version` function. ({issue}`4627`)
+* Allow prepared statement parameters for `SHOW STATS`. ({issue}`6582`)
+* Update tzdata version to 2020d. As a result, queries can no longer reference the 
+  `US/Pacific-New` zone, as it has been removed. ({issue}`6660`)
+* Improve performance of queries using {func}`rank()` window function. ({issue}`6333`)
+* Improve performance of {func}`sum` and {func}`avg` for `decimal` types. ({issue}`6951`)
+* Improve join performance. ({issue}`5981`)
+* Improve query planning time. ({issue}`6544`)
+* Fix window and streaming aggregation semantics regarding peer rows. Now peer rows are
+  not distinct from each other, while previously they had to be equal. ({issue}`6472`) 
+* Fix query failure when using element of `array(timestamp(p))` in a complex expression 
+  for `p` greater than 6. ({issue}`6350`)
+
+## Web UI
+
+* Support JWK with certificate chain for OAuth2 authorization. ({issue}`6428`)
+* Optionally verify JWT audience (`aud`) field for OAuth2 Web UI authentication. ({issue}`6501`)
```

Comment:
What is optionality driven by? Should we reference config property here?

Reply:
@lukasz-walkiewicz @kokosing

Reply:
Config property is here: `http-server.authentication.oauth2.audience`, if specified then we require token to have this audience set.

Currently, we are still missing docs for entire feature. We are going to add it soon. 

FYI @s2lomon @lukasz-walkiewicz

Reply:
Same, this doesn't seem specific to web UI

Reply:
Ok, I'll move it to a Security section